pkg/services/authn/authnimpl/sync/oauth_token_sync.go

```diff
@@ -84,11 +87,14 @@ func (s *OAuthTokenSync) SyncOauthTokenHook(ctx context.Context, identity *authn
 	currentOAuthInfo := s.socialService.GetOAuthInfoProvider(provider)
 	if currentOAuthInfo == nil {
 		s.log.Warn("OAuth provider not found", "provider", provider)
+
+		s.cache.Set(identity.ID, struct{}{}, maxOAuthTokenCacheTTL)
```

Comment:
I'm a bit puzzled about this one. What would be this case? THe user has an `OAuthEntry` but it's comes from no known provider?

Reply:
This could happen if user has a entry to a oauth provider that used to be configured but no longer are

Reply:
Wow, thanks, talking about an edge case :smile: